nservicebus/upgrades/asp-saga-deduplication.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Patch Azure Storage Persistence Sagas
+summary: Instructions on how to patch Azure Storage Persistence when saga duplication occurs.
+tags:
+ - upgrade
+related:
+ - nservicebus/sagas
+ - nservicebus/azure/azure-storage-persistence
+---
+
+## Summary
+
+This document explains how to upgrade and patch a system for [Azure Storage Persistence bug #634](https://github.com/Particular/PlatformDevelopment/issues/634) using the NServiceBus Azure hotfix release xxxxxxx.
+
+### How to know if a system is affected
+This bug will affect a system if both of the following conditions are met:
+- more than one IAmStartedBy<T> for a given saga
+- concurrency is set to > 1
+
+### Symptoms
+The following are three of the symptoms that a system may exhibit as result of this bug.
```

Comment:
all these symptoms are pretty vague. are you actually trying to explain what causes the bug?